checks/pinned_dependencies.go

```diff
@@ -52,6 +52,23 @@ type gitHubActionWorkflowConfig struct {
 	Name string `yaml:"name"`
 }
 
+// stringWithLine is for when you want to know the line number of a field in a YAML file.
```

Comment:
since we want this line also for Dockerfiles, etc let's remove the end of the comment.

Reply:
Comment updated